components/Item/Metadata/ImageEditor.vue

```diff
@@ -127,11 +127,13 @@ export default Vue.extend({
         })
       ).data;
     },
-    imageFormat(imageInfo: ImageInfo): string {
-      return this.getImageUrlForElement(imageInfo.ImageType as ImageType, {
-        itemId: this.metadata.Id,
-        tag: imageInfo.ImageTag as string
-      });
+    imageFormat(imageInfo: ImageInfo): string | undefined {
+      if (imageInfo.ImageType) {
+        return this.getImageUrlForElement(imageInfo.ImageType, {
+          itemId: this.metadata.Id,
+          tag: imageInfo.ImageTag as string
```

Comment:
I don't know much this API, but is there a reason for not checking in the if the `ImageTag`?

Reply:
Good catch

Reply:
Another thing to refactor later on haha